packages.yml

```diff
@@ -1,3 +1,5 @@
 packages:
-- package: fivetran/intercom_source
-  version: [">=0.4.0", "<0.5.0"]
+#- package: fivetran/intercom_source
+#  version: [">=0.5.0", "<0.6.0"]
+  - git: "https://github.com/fivetran/dbt_intercom_source.git" # git URL
+    revision: feature/postgres-comp # tag or branch name
```

Comment:
@fivetran-joemarkiewicz Ran into this issue when attempting to merge. Do i run `git pull` as suggested to resolve?
<img width="836" alt="Screen Shot 2022-07-26 at 9 08 40 AM" src="https://user-images.githubusercontent.com/108772760/181056667-805f184b-d8f0-40fc-80c0-2b463baf3293.png">

Reply:
@fivetran-avinash this error is because v0.5.0 of the source package is not live yet. The comment is simply as a reminder before we merge the branch to `main`. For the time being you will want to keep the git version. You won't actually want to make this change until the source package changes are live on the hub.

Reply:
@fivetran-joemarkiewicz reverted for now!